src/screens/ModelPage/ModelPage.js

```diff
@@ -92,206 +61,42 @@ function ModelPage() {
-          <MapWrapper visible={mapOption === MAP_FILTERS.NATIONAL}>
-            <Map
-              hideLegend={true}
-              setMapOption={setMapOption}
-              setMobileMenuOpen={setMobileMenuOpen}
-            />
-          </MapWrapper>
-
-          {stateId !== MAP_FILTERS.DC && (
-            <CountyMapAltWrapper visible={mapOption === MAP_FILTERS.STATE}>
-              <CountyMap
-                fill={
-                  projections.primary
-                    ? projections.getAlarmLevelColor()
-                    : '#e3e3e3'
-                }
-                stateSummary={stateSummary}
-                selectedCounty={selectedCounty}
-                setSelectedCounty={fullFips => {
-                  const county = _.find(
-                    US_STATE_DATASET.state_county_map_dataset[stateId]
-                      .county_dataset,
-                    ['full_fips_code', fullFips],
-                  );
-
-                  goTo(
-                    `/us/${stateId.toLowerCase()}/county/${
-                      county.county_url_name
-                    }`,
-                  );
-                  setMobileMenuOpen(false);
-                  setSelectedCounty(county);
-                }}
-              />
-            </CountyMapAltWrapper>
-          )}
-        </MapContentInner>
-      </MapContentWrapper>
-    );
-  };
 
   return (
     <Wrapper>
-      {metaTags}
+      <AppMetaTags
+        canonicalUrl={`/us/${stateId.toLowerCase()}`}
+        pageTitle={`${stateName} Forecast`}
+        pageDescription={actionTitle}
+        shareTitle={actionTitle}
+        shareDescription={actionDescription}
+      />
       <ContentWrapper>
-        {renderHeader()}
-        {projections && !projections.primary ? (
-          <div>
-            <StyledNoResultsWrapper>
-              <StyledNoResults>
-                <div>
-                  We either weren't able to get data for {selectedCounty.county}
-                  , {selectedCounty.state_code}, or no cases have yet been
-                  confirmed.
-                </div>
-                <div>
-                  We’re unable to produce our model for this county until we
-                  have data. Check back soon.
-                </div>
-                <div style={{ marginTop: '1rem' }}>
-                  View projections for{' '}
-                  <span onClick={() => goTo('/us/' + stateId.toLowerCase())}>
-                    {stateName}
-                  </span>
-                </div>
-              </StyledNoResults>
-            </StyledNoResultsWrapper>
-            {renderMapContent()}
-          </div>
-        ) : (
-          [renderMainContent(), renderMapContent()]
-        )}
+        <SearchHeaderWrapper>
+          <SearchHeader
+            setMapOption={setMapOption}
+            mobileMenuOpen={mobileMenuOpen}
+            setMobileMenuOpen={setMobileMenuOpen}
+          />
+        </SearchHeaderWrapper>
+        <ChartsHolder
```

Comment:
reading this I'm not totally sure what a "chartsholder" is.  i guess it holds some charts?  It's weird that it contains the page header in it since that has nothing to do with charts..

also i'd consider renaming ModelPage to "LocationPage"

Reply:
cool  ya this seems out of scope but i changed to locationpage

Reply:
i got a lil nervus and left it mostly the same since i was changing so many other things. made https://github.com/covid-projections/covid-projections/issues/598 to address in the future